simpa/utils/libraries/tissue_library.py

```diff
@@ -12,27 +12,40 @@
 from simpa.utils.calculate import randomize_uniform
 from simpa.utils.libraries.spectrum_library import AbsorptionSpectrumLibrary
 
+from typing import Union, List
+import torch
+
 
 class TissueLibrary(object):
     """
-    TODO
+    A library, returning molecular compositions for various typical tissue segmentations.
     """
 
-    def get_blood_volume_fractions(self, total_blood_volume_fraction=1e-10, oxygenation=1e-10):
+    def get_blood_volume_fractions(self, oxygenation: Union[float, int, torch.Tensor] = 1e-10,
```

Comment:
Since you swapped the parameters, did you check all use-cases of this functions that the args are set for the correct parameters? If not, please check that

Reply:
All checked :)